Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -127,7 +183,18 @@ fun ConversationContent(
             .exclude(WindowInsets.ime),
         modifier = modifier.nestedScroll(scrollBehavior.nestedScrollConnection)
     ) { paddingValues ->
-        Column(Modifier.fillMaxSize().padding(paddingValues)) {
+        Column(
+            Modifier.fillMaxSize().padding(paddingValues)
+                .background(color = background)
+                .border(width = 2.dp, color = borderStroke)
+                .dragAndDropTarget(shouldStartDragAndDrop = { event ->
+                    event
+                        .mimeTypes()
+                        .contains(
+                            ClipDescription.MIMETYPE_TEXT_PLAIN
```

Comment:
Could it also support images?

Reply:
Yes, it could :) What sample should be the one providing the image?

Reply:
any i guess :) happy for this to merge without it for now though.

Reply:
Would it be ok to do it in a subsequent PR? So I can investigate further how to better share the image?

Reply:
yes sure, i approved it